### PR TITLE
fix(indexer): Retry when tx receipt not found

### PR DESCRIPTION
**Pull request by classicalliu**
Retry for about 10 times(~55s), if sill not found, return error

### CODE REVIEW

File: crates/indexer/src/indexer.rs [hunk 1]
```diff
@@ -519,19 +519,39 @@ impl Web3Indexer {
...
+                        tx_hash_hex,
+                        block_number,
+                        tx_index
+                    )
+                })?
+                .into();
+            Ok(tx_receipt)
+        };
+
+        let retry_count = 10;
```
Comment (RetricSu) [inline crates/indexer/src/indexer.rs:539]:
```suggestion
        let max_retry = 10;
```

File: crates/indexer/src/indexer.rs [hunk 2]
```diff
@@ -519,19 +519,39 @@ impl Web3Indexer {
...
+        let mut retry_times = 0;
```
Comment (RetricSu) [inline crates/indexer/src/indexer.rs:540]:
```suggestion
        let mut retry_count = 0;
```

Comment (RetricSu) [reply crates/indexer/src/indexer.rs:540]: rename the variables?

File: crates/indexer/src/indexer.rs (hunk 1 above)
Comment (classicalliu) [reply crates/indexer/src/indexer.rs]: Updated

**Review by RetricSu — APPROVED** [review]

**Pull request merged**